src/rpc/parsers.ts

```diff
@@ -26,6 +26,32 @@ export function parseRawSendTransaction(
   return { ...r } as Api.BaseSendTransactionResponse;
 }
 
+export function parseRawTransactions(
+    r: Api.RawTransactionInfo
+): Api.TransactionInfo {
+  const info : Api.TransactionInfo = {
+    status: r.status,
+    ledger: r.ledger!,
+    createdAt: r.createdAt!,
+    applicationOrder: r.applicationOrder!,
+    feeBump: r.feeBump!,
+    envelopeXdr: xdr.TransactionEnvelope.fromXDR(r.envelopeXdr!, 'base64'),
+    resultXdr: xdr.TransactionResult.fromXDR(r.resultXdr!, 'base64'),
+    resultMetaXdr: xdr.TransactionMeta.fromXDR(r.resultMetaXdr!, 'base64'),
+  }
```

Comment:
Can you combine this with [Server.getTransaction](https://github.com/stellar/js-stellar-sdk/blob/aaf3360c00e1d7b1617717e635db491f84036ba0/src/rpc/server.ts#L438)? There is a lot of shared code.